core/util/createBitmaps.js

```diff
@@ -129,6 +129,9 @@ function delegateScenarios (config) {
 
   if (config.engine.startsWith('puppet')) {
     return pMap(scenarioViews, runPuppet, { concurrency: asyncCaptureLimit });
+  }
+  else if (/chrom./i.test(config.engine)) {
+    logger.error(`Engine "${(typeof config.engine === 'string' && config.engine) || 'undefined'}" not recognized! Chromy is no longer supported in version 5+. Please use version 4.x.x for chromy support.`);
```

Comment:
Maybe this should just be: "Chromy is no longer supported in version 5+. Please use version 4.x.x for Chromy support."  I don't think you need the first bit with all the type checking and stuff.  We already know inside this block that we have a chromy user who probably bumped their version and now they are confused about why everything is failing.  Please let me know if there is something I am missing here.  Thank you!

Reply:
updated